### ISSUE TITLE
Further Development

### ISSUE THREAD

**Issue by doc-halo** [K-3D/k3d#45]
Is there any interest in kickstarting development back up?

I'm an educator and hobbyist animator, and am looking for a simple 3d animation program to introduce to high school students. And to use myself for my projects.

I've been using Blender for years but it has way too many bells and whistles that I never use. And when I showed Blender to my teenager, the interface was way too daunting for him. I need something simpler. My only option right now for animating is K-3d and Art of Illusion, but k-3d has the more intuitive interface.

I'm a non-coder, but I'm willing to contribute a lot of my time in other areas if there is interest from coders.

Thanks,

**Comment by kenjennings** [comment]
I agree with you.   I would like a simple modeler.   I had used programs
like this to do 3D illustrations for work.  Then the simple tools
disappeared from the distros I use.   Blender is rocket science.

**Comment by doc-halo** [comment]
I have experience with all things writing: technical writing, docs, making tutorials & videos.

I can also help with recruiting developers and users, and getting the word out about the project. 

I'll be even willing to learn some basic programming stuff to help out.

**Comment by barche** [comment]
It seems that somewhere along the way my email notifications for this repo got turned off. As is obvious from the state of things, K-3D is neglected pretty badly. I got kind of discouraged while porting to GTK3 (see gtk3 branch). Anyone is welcome to contribute, but so far nobody has shown much motivation to pick this up.
